[BUG] Failed Build due to missing dependency

### Description

I was attempting to build my own .jar just to check it would build normally **before** i added i patch for my specific server. i switched to both the hikari and release branches and recieved the same error during building. i checked all the issues and pinned issues and found nothing, this is the error i got:

```
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  45.150 s
[INFO] Finished at: 2025-03-13T14:14:24Z
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal on project quickshop-bukkit: Could not collect dependencies for project com.ghostchu:quickshop-bukkit:takari-jar:6.2.0.8
[ERROR] Failed to read artifact descriptor for com.tcoded:FoliaLib:jar:0.3.4
[ERROR] 	Caused by: The following artifacts could not be resolved: com.tcoded:FoliaLib:pom:0.3.4 (absent): Could not transfer artifact com.tcoded:FoliaLib:pom:0.3.4 from/to devmart-other (https://nexuslite.gcnt.net/repos/other/): status code: 526, reason phrase:  (526)
[ERROR] 
```

any help on this is appreciated, Thank you


### Steps to reproduce

1. build the jar using maven without editing the code from github
2. receive error

### Expected Behaviour

Quickshop should've builded successfully 

### Screenshots

![Image](https://github.com/user-attachments/assets/f77429ab-eb6e-410b-b630-be31d24baf4d)

### `/quickshop paste` URL

quickshop did not build

### Additional Context

_No response_

### Checklist

- [x] I'm running Paper or Spigot, and not a fork
- [x] I confirm that Paper/Spigot has been updated to the latest build
- [x] I confirm that QuickShop-Hikari has been updated to the latest stable version released on Modrinth (or the latest CI version)
- [ ] I confirm that I have not read these checkboxes and therefore I just ticked them all.
- [x] I confirm that I'm using QuickShop-Hikari, not QuickShop-Reremake, and I'm well aware that they're maintained by different people, and that Reremake issues shouldn't be reported here.
- [x] I confirm that I am running a server that is not a Hybird Server, (e.g. Mohist, Magma, CatServer, Banner, etc.), and I am aware that QuickShop-Hikari may not function properly on a Forge/Fabric hybrid server, and I am running at my own risk on such a server program, and I am aware that the I run such server-side programs at my own risk and know that I will not receive any support or help for this behavior.
- [x] I am well aware that if the Issue Ticket is not filled out correctly and completely, it will simply be closed without any response or reason.

--- a/quickshop-bukkit/pom.xml
+++ b/quickshop-bukkit/pom.xml
@@ -74,10 +74,6 @@
             <id>sonatype-oss-snapshots1</id>
             <url>https://s01.oss.sonatype.org/content/repositories/snapshots/</url>
         </repository>
-        <repository>
-            <id>devmart-other</id>
-            <url>https://nexuslite.gcnt.net/repos/other/</url>
-        </repository>
     </repositories>
     <dependencies>
         <dependency>
@@ -377,9 +373,9 @@
             <version>1.26.2</version>
         </dependency>
         <dependency>
-            <groupId>com.tcoded</groupId>
+            <groupId>com.github.technicallycoded</groupId>
             <artifactId>FoliaLib</artifactId>
-            <version>0.3.4</version>
+            <version>main-SNAPSHOT</version>
             <scope>compile</scope>
         </dependency>
         <dependency>

